### DISCUSSION TITLE
关于yacd面板里面overview页面的Upload Total和Download Total清零疑问

### DISCUSSION THREAD

**Discussion by Jefffish09** [vernesong/OpenClash#2717] [Q&A] [answered]
想问下，yacd面板里面overview页面的Upload Total和Download Total会在什么情况重置为零（清零）？

是每天凌晨自动清零？还是openclash重启时自动清零？

一直没看出规律。

![image](https://user-images.githubusercontent.com/44800122/191930607-548944c9-439f-4287-95f7-f2a5cd8cd452.png)

**Comment by vernesong** [comment, marked as answer]
内核重启

**Reply by Jefffish09** [reply]
这个内核，是指clash内核，还是openwrt系统内核？

**Reply by vernesong** [reply]
clash

**Reply by Jefffish09** [reply]
明白了，感谢！
